evoting/service/service.go

```diff
@@ -174,6 +179,47 @@ func (s *Service) Login(req *evoting.Login) (*evoting.LoginReply, error) {
 	return &evoting.LoginReply{Token: token, Admin: admin, Elections: elections}, nil
 }
 
+// LookupSciper calls https://people.epfl.ch/cgi-bin/people/vCard?id=sciper
+// to convert scipers to names
+func (s *Service) LookupSciper(req *evoting.LookupSciper) (*evoting.LookupSciperReply, error) {
+	url := fmt.Sprintf("https://people.epfl.ch/cgi-bin/people/vCard?id=%s", req.Sciper)
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
Please wrap resp.Body in an io.LimitReader with a limit of 100k to prevent using too much RAM in case people.epfl.ch loses it's mind and spams us.